src/ResponseTagger.php

```diff
@@ -123,7 +123,7 @@ public function addTags(array $tags)
             throw new InvalidTagException('Empty tags are not allowed');
         }
 
-        $this->tags = array_merge($this->tags, $filtered);
+        $this->tags = array_unique(array_merge($this->tags, $filtered));
```

Comment:
i am pondering whether we should do the unique only at the end when building the tags header. not sure whats better performance - doing unique only at the end in getTagsHeaderValue would add less overhead for people who's system is not adding duplicate tags. on the other hand, a system with massive duplicates would profit from a rolling de-duplication. also depends on whether addTags is called a lot or only a few times... wdyt?

Reply:
I asked myself the same question but in the end I guess it's such a minor thing, nobody will ever notice 😄 I put it into the `addTags()` because there's already some filtering going on there. I just figured it would be better to add it there so it's all in one place. But again, at the end of the day it doesn't really matter I think 😊

Reply:
yeah, agreed